docs/kusama-parameters.md

```diff
@@ -1,57 +1,74 @@
 | Voting period | 1 day | 14,400 | The council's voting period for motions.                             |
 
+The Kusama Council consists of up to 19 members and up to 19 runners-up.
+
 | Technical committee     | Time    | Slots   | Description                                                                                    |
 | ----------------------- | ------- | ------- | ---------------------------------------------------------------------------------------------- |
 | Cool-off period         | 7 days  | 604,800 | The time a veto from the technical committee lasts before the proposal can be submitted again. |
 | Emergency voting period | 3 hours | 1,800   | The voting period after the technical committee expedites voting.                              |
 
 ### Staking, Validating, and Nominating
 
-| Kusama               | Time    | Slots   | Description                                                                                                                                     |
-| -------------------- | ------- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
-| Term duration        | 6 hours | 3,600   | The time for which a validator is in the set after being elected. Note, this duration can be shortened in the case that a validator misbehaves. |
-| Nomination period    | 6 hours | 3,600   | Every 6 hours, a new validator set is elected according to Phragmen's method.                                                                   |
-| Bonding duration     | 7 days  | 604,800 | How long until your funds will be transferrable after unbonding.                                                                                |
-| Slash defer duration | 7 days  | 604,800 | Prevents overslashing and validators "escaping" and getting their nominators slashed with no repercussions to themselves                        |
+| Kusama               | Time    | Slots   | Description                                                                                                                                                                                         |
+| -------------------- | ------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Term duration        | 6 hours | 3,600   | The time for which a validator is in the set after being elected. Note, this duration can be shortened in the case that a validator misbehaves.                                                     |
+| Nomination period    | 6 hours | 3,600   | How often a new validator set is elected according to Phragmen's method.                                                                                                                            |
```

Comment:
```suggestion
| Nomination period    | 6 hours | 3,600   | How often a new validator set is elected according to Phragmén's method.                                                                                                                            |
```

Reply:
Phragmen should be Phragmén, I'm not sure the commit suggestion there includes that